From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Tue, 4 Aug 2026 20:03:36 +0200
Subject: dt-bindings: mfd: x-powers,ac100: Allow jack-type in the codec node

Same property, same meaning as on allwinner,sun8i-a33-codec: it names
what the board's jack accepts, and its absence disables detection. The
codec sub-node rejects unknown properties, so it has to be listed
before a DTS can use it.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 Documentation/devicetree/bindings/mfd/x-powers,ac100.yaml | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Documentation/devicetree/bindings/mfd/x-powers,ac100.yaml b/Documentation/devicetree/bindings/mfd/x-powers,ac100.yaml
index 111111111111..222222222222 100644
--- a/Documentation/devicetree/bindings/mfd/x-powers,ac100.yaml
+++ b/Documentation/devicetree/bindings/mfd/x-powers,ac100.yaml
@@ -37,6 +37,14 @@ properties:
       "#sound-dai-cells":
         const: 1
 
+      jack-type:
+        enum:
+          - headset
+          - headphone
+        description:
+          Type of accessory the board's audio jack accepts. If not
+          specified, jack detection is disabled.
+
       LDOIN-supply: true
       AVCC-supply: true
       VDDIO1-supply: true
-- 
Armbian

